pkg/flowkit/arguments.go

```diff
@@ -158,19 +158,25 @@ func ParseArgumentsWithoutType(fileName string, code []byte, args []string) (scr
 		astType := parameterList[index].TypeAnnotation.Type
 		semaType := checker.ConvertType(astType)
 
-		switch semaType {
-		case sema.StringType:
-			if len(argumentString) > 0 && !strings.HasPrefix(argumentString, "\"") {
-				argumentString = "\"" + argumentString + "\""
+		for {
```

Comment:
why is this for loop needed? it breaks on the first loop in any case

Reply:
Looping 2 times if type is Optional. I could have checked if optional type and unwrap the type before switch. I can change to that if it is more readable.

Reply:
ohh completely missed that `continue`. Makes sense it's ok that way.